packages/api/test/nfts-store.spec.js

```diff
@@ -94,19 +95,22 @@ test('should store image', async (t) => {
   ])
 })
 
-// Miniflare doesn't currently preserve pathnames in File objects uploaded via FormData.
-// TODO: re-enable this test once https://github.com/cloudflare/miniflare/pull/309 is merged
-test.skip('should store dir wrapped image', async (t) => {
+test('should store dir wrapped image', async (t) => {
   const config = getTestServiceConfig(t)
   const mf = getMiniflareContext(t)
   const { token, userId } = await createTestUser(t)
 
   const metadata = {
     name: 'name',
     description: 'stuff',
-    image: new File(['fake image'], '/dir/cat.png', { type: 'image/png' }),
+    // Note that the multipart form parser used by miniflare does not remove
```

Comment:
did you report this upstream ?

Reply:
good idea :smile: https://github.com/cloudflare/miniflare/issues/352

Also made one about the FormData problem: https://github.com/cloudflare/miniflare/issues/351